projects.json

```diff
@@ -2227,6 +2227,10 @@
       {
         "version": "4.0",
         "commit": "549d4c97664bea31af61a4d3d6184dc3e970880a"
```

Comment:
@jpsim Thanks for the update! We'll also need to have the 4.0 commit updated, or else that config will still fail for the same issue as before. That new commit (60f98ec5) should work for both 4.0 and 4.2 configs.

Reply:
Ah yes, done.